pkgs/by-name/qq/qqsp/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  adwaita-qt,
+  libadwaita,
+  adw-gtk3,
+  gst_all_1,
+  gnome-themes-extra,
+  librsvg,
+  fetchgit,
+  qt5,
+}:
+stdenv.mkDerivation rec {
+  name = "qqsp";
+  version = "1.9";
+  src = fetchFromGitHub {
+    owner = "Sonnix1";
+    repo = "Qqsp";
+    rev = "v${version}";
+    sha256 = "eDgoa+/dcJ8Ti+YLHgKUKus0+zRrFEuJ19wUpbFpcBU=";
+  };
+  buildInputs = [
+    adwaita-qt
+    libadwaita
+    adw-gtk3
+    gnome-themes-extra
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+  ];
+  nativeBuildInputs = with qt5; [
+    qmake
+    wrapQtAppsHook
+    qtmultimedia
+    qtwebengine
+    librsvg
+  ];
+  installPhase = ''
+    install -D Qqsp $out/bin/qqsp
+    install -D $src/icons/qsp-logo-vector.svg $out/icons/scalable/apps/qsp.svg
+    install -D $src/Qqsp.desktop $out/share/applications/Qqsp.desktop
+    install -D $src/qsp.mime $out/share/mime/packages/qsp.xml
+    for i in 16 24 32 48 64 96 128 256 512; do
+      mkdir -p $out/share/icons/''${i}x$i/apps
+      rsvg-convert -w $i -h $i -f png $src/icons/qsp-logo-vector.svg -o $out/share/icons/''${i}x$i/apps/qsp.png
+    done
+  '';
+  meta = {
+    description = "Qt Quest Soft Player";
+    license = lib.licenses.mit;
```

Comment:
Missing fields: `homepage`, `platforms`

Reply:
Is the git good enough for homepage? It doesn't have an official website as far as I know, just the git and some unofficial posts on external forums/wikis.

Reply:
That's perfectly fine, lots of packages already do that

Reply:
Great, I think that should be everything resolved then